test/EFCore.Cosmos.FunctionalTests/VectorSearchCosmosTest.cs

```diff
@@ -0,0 +1,537 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.ComponentModel.DataAnnotations.Schema;
+using Microsoft.Azure.Cosmos;
+using Microsoft.EntityFrameworkCore.Cosmos.Extensions;
+using Microsoft.EntityFrameworkCore.Cosmos.Internal;
+
+namespace Microsoft.EntityFrameworkCore;
+
+public class VectorSearchCosmosTest : IClassFixture<VectorSearchCosmosTest.ReadItemFixture>
+{
+    public VectorSearchCosmosTest(ReadItemFixture fixture, ITestOutputHelper testOutputHelper)
+    {
+        Fixture = fixture;
+        this.testOutputHelper = testOutputHelper;
+        fixture.TestSqlLoggerFactory.Clear();
+    }
+
+    protected ReadItemFixture Fixture { get; }
+
+    private readonly ITestOutputHelper testOutputHelper;
+
+    [ConditionalFact]
+    public virtual async Task Query_for_vector_distance_sbyte_array()
+    {
+        await using var context = CreateContext();
+        var inputVector = new sbyte[] { 2, 1, 4, 6, 5, 2, 5, 7, 3, 1 };
```

Comment:
nit: I've personally switched to collection initializers:

```suggestion
        sbyte[] inputVector = [2, 1, 4, 6, 5, 2, 5, 7, 3, 1];
```